features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListenerTest.kt

```diff
@@ -127,6 +127,16 @@ internal class JankStatsActivityLifecycleListenerTest {
         verify(mockJankStatsProvider).createJankStatsAndTrack(mockWindow, testedJankListener, mockInternalLogger)
     }
 
+    @Test
+    fun `M not track same activity twice W onActivityStarted {}`() {
+        // When
+        testedJankListener.onActivityStarted(mockActivity)
+        testedJankListener.onActivityStarted(mockActivity)
+
+        // Then
+        assertThat(testedJankListener.activeActivities[mockWindow]?.size).isEqualTo(1)
```

Comment:
if I'm not wrong, should work also with nullable type

```suggestion
        assertThat(testedJankListener.activeActivities[mockWindow]).hasSize(1)
```